Project.toml

```
@@ -20,7 +21,7 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [compat]
+GraphMakie = "0.5.5"
 MINDFul = "0.2"
 NestedGraphMakie = "0.1"
-GraphMakie = "0.5.5"
 julia = "1.9"
```

Can you add julia 1.10 here ?
```suggestion
julia = "1.9, 1.10"
```